client/router.js

```diff
@@ -148,11 +148,11 @@ class Router {
       const queryParams = this._qs.parse(context.querystring);
       this._current = {
         path: context.path,
-        context: context,
+        context,
```

Comment:
👍

Reply:
also my linter. Damn.

Reply:
This ES6 I like, right way to do so, this lib was ES5, so I haven't changed all the code (yet)